writing_center/cron/__init__.py

```diff
@@ -47,32 +47,24 @@ def send_reminder_emails(self):
         try:
             upcoming_appointments = self.cron.get_upcoming_appointments()
             for appointment in upcoming_appointments:
-                student = self.cron.get_user(appointment.student_id)
                 tutor = self.cron.get_user(appointment.tutor_id)
+                student = self.cron.get_user(appointment.student_id)
+                subject = "Your Upcoming Appointment"
```

Comment:
I don't love the subject name here. I think the previous subject name was better suited - "Writing Center Multilingual Appointment Reminder" and "Writing Center Appointment Reminder"

Reply:
done